lib/sm/client.rb

```diff
@@ -108,6 +107,8 @@ def raise_not_authenticated
     def connection
       @connection ||= Faraday.new(@config.base_path, headers: BASE_REQUEST_HEADERS, request: request_options) do |conn|
         conn.use :breakers
+        conn.request :camelcase
+        conn.request :multipart_request
```

Comment:
these middlewares replace the nasty "normalize_and_jsonify" method we were using previously.